packages/xo-remote-parser/src/index.js

```diff
@@ -77,6 +78,22 @@ export const parse = string => {
     object.username = decodeURIComponent(parsed.username)
     object.password = decodeURIComponent(parsed.password)
     object = { ...parseOptionList(parsed.query), ...object }
+  } else if (type === 'azure') {
+    object.type = 'azure'
+    let username, password, host, port, path, optionList
+    // Some users have a remote with a colon in the URL, which breaks the parsing since this commit: https://github.com/vatesfr/xen-orchestra/commit/fb1bf6a1e748b457f2d2b89ba02fa104554c03df
+    try {
+      ;[, username, password, host, port, path, optionList] = AZURE_RE.exec(rest)
```

Comment:
can you retest with a url created by the format method
If it still doesn't work, please add a comment of why you don't use urlparser